backend/main-service/src/main/java/kkakka/mainservice/coupon/domain/Discount.java

```diff
@@ -0,0 +1,80 @@
+package kkakka.mainservice.coupon.domain;
+
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import kkakka.mainservice.category.domain.Category;
+import kkakka.mainservice.product.domain.Product;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Getter
+@Table(name = "Discount")
+@AllArgsConstructor
+@NoArgsConstructor
+public class Discount {
+
+    @Id
+    @GeneratedValue
+    @Column(name = "id")
+    private Long id;
+
+    @OneToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "category_id")
+    private Category category;
+
+    @OneToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "product_id")
+    private Product product;
+
+    private String name;
+    private Integer discount;
+    private LocalDateTime startedAt;
+    private LocalDateTime expiredAt;
+    private boolean isDeleted;
```

Comment:
```suggestion
    private Boolean isDeleted;
```
엔티티 필드는 모두 래퍼 클래스로 하기로 했던걸로 기억합니당
사실 여기서는 아니어도 상관 없을거같긴 한데...

Reply:
통일성있게 수정하도록 하겠습니다!